src/getSchema.js

```diff
@@ -17,10 +18,40 @@ export default function getSchema (schemaPath: string): GraphQLSchema {
     return buildASTSchema(parse(source))
   } catch (error) {
     throw new Error(`
-Error loading schema. Expected the schema to be a .graphql file using the
-GraphQL schema definition language. Error detail:
+Error loading schema from .graphql file. Error detail:
 
 ${error.stack}
     `.trim())
   }
 }
+
+function parseJSON(schemaPath: string): GraphQLSchema {
+  try {
+    let source = JSON.parse(fs.readFileSync(schemaPath, 'utf8'));
+    if (typeof source !== 'object' || !('data' in source)) {
```

Comment:
I am pretty new to GraphQL so I can't say with certainty this always is in .data but it sure is for github.